sdk-setup/src/mb2

```diff
@@ -2672,6 +2679,12 @@ run_build() {
     local hook_exports=()
     local rpms=$(find_rpms "$OPT_OUTPUTDIR" "$OPT_PKGDIR" "$STATEDIR/spec")
     run_hook post-package "${hook_exports[*]}" ${rpms} || fatal "The 'post-package' hook failed"
+
+    if [[ ! $BUILD_NORPMLINT ]]; then
+        CHECK_SUITES="$CHECK_RPMLINT"
+        run_check__process_args_helper "" "$CHECK_SUITES" ""
+        run_check --already-locked || return 0
```

Comment:
I would rather use `return 0` as the last statement of the function, so that more steps can be added in future without changing this line.

Reply:
fixed